packages/web-pkg/tests/unit/components/sidebar/Details/SpaceDetails.spec.ts

```diff
@@ -1,4 +1,4 @@
-import SpaceDetails from '../../../../../src/components/SideBar/Details/SpaceDetails.vue'
+import SpaceDetails from 'web-pkg/src/components/SideBar/Details/SpaceDetails.vue'
```

Comment:
Should fix the pipeline error for now, I think. We should rename `sideBar` to `SideBar`, but then we need to adjust a lot of imports... So maybe better to do this as a follow-up.

```suggestion
import SpaceDetails from 'web-pkg/src/components/sideBar/Details/SpaceDetails.vue'
```

Reply:
aaaah damn thx, thought i lost my mind